scripts/remote-deploy.sh

```diff
@@ -27,5 +27,8 @@ if [ -n "$REVISION_DUMMY_API" ]; then
     sed -i -e "s/^REVISION_DUMMY_API=.*$/REVISION_DUMMY_API=$REVISION_DUMMY_API/g" .env
 fi
 
+# avoid nginx+fpm shared volumes persisting files from older releases
+# assume all service are stateless
```

Comment:
Unclear how data in the demo-like environments is going to work.

Reply:
Being a demo it should be recreatable from scratch, until we need to store state produced by users like exploratory tests. An `unstable` environment will require cleanup often

Reply:
Sounds like we'd want on-demand temporary instances for that case.